packages/perf-benchmarks/scripts/format-results.mjs

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+/**
+ * Forms the JSON results from multiple Tachometer runs and outputs a Markdown table of results.
+ */
+
+import { markdownTable } from 'markdown-table';
+import fs from 'fs/promises';
+
+function avg(a, b) {
+    return (a + b) / 2;
+}
+
+function fmt(num) {
+    return num.toFixed(2);
+}
+
+async function main() {
+    const jsonFiles = [...process.argv].filter((_) => _.endsWith('.json'));
+    const header = [
+        'Benchmark',
+        'Before (low)',
+        'Before (high)',
+        'Before (avg)',
+        'After (low)',
+        'After (high)',
+        'After (avg)',
+        'Delta (low)',
+        'Delta (high)',
+        'Delta (avg)',
+        'Delta perc (low)',
+        'Delta perc (high)',
+        'Delta perc (avg)',
+    ];
+
+    const results = await Promise.all(
+        jsonFiles.map(async (file) => {
+            const json = JSON.parse(await fs.readFile(file, 'utf8'));
+            const { benchmarks } = json;
+            const { low: deltaAbsLow, high: deltaAbsHigh } = benchmarks[0].differences[1].absolute;
+            const { low: deltaPercLow, high: deltaPercHigh } =
+                benchmarks[0].differences[1].percentChange;
+            const { low: beforeLow, high: beforeHigh } = benchmarks[1].mean;
```

Comment:
What does `mean.low` and `mean.high` represents? Is it the mean +/- the standard deviation?

Reply:
From the [Tachometer readme](https://github.com/Polymer/tachometer):

> The ranges tachometer reports are 95% confidence intervals for the percent change from one benchmark to another.

I take the average of the low and high because I find it's much easier to think about one number rather than two.